### PR TITLE
Setup a test travis cron job

### PR DESCRIPTION
**Pull request by bmac**
I trigger a travis job off of this branch using the cron API. Here is a link to the successful job. https://travis-ci.org/bocoup/test262-automation/jobs/394229534

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "test262-automation": "index.js"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"No tests specified yet\" && exit 0",
```
Comment (bmac) [inline package.json]: The test command does nothing for now. I just wanted to prove we could have a test target alongside the cron task.

File: cron.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+BRANCH="test-branch-`date +%Y-%m-%d`"
+git checkout -B $BRANCH
+touch update-log.txt
+echo "Updated at `date`" >> update-log.txt
+git add update-log.txt
+git commit -m "cron update"
+git push https://$GITHUB_TOKEN@github.com/bocoup/test262-automation.git +$BRANCH
+git checkout -
```
Comment (bmac) [inline cron.sh]: This is a simple proof of concept task that creates a new branch from the travis cron process.

File: cron.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+BRANCH="test-branch-`date +%Y-%m-%d`"
+git checkout -B $BRANCH
+touch update-log.txt
+echo "Updated at `date`" >> update-log.txt
+git add update-log.txt
+git commit -m "cron update"
+git push https://$GITHUB_TOKEN@github.com/bocoup/test262-automation.git +$BRANCH
```
Comment (bmac) [inline cron.sh]: I've configured travis to inject the `GITHUB_TOKEN` environment variable for the jobs in this repo.  The token is currently set to be a github oauth token for the @test262-automation account.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - "10"
+
+jobs:
+  include:
+    - script: npm run test
+    - script: ./cron.sh
+      if: type IN (cron)
+
```
Comment (nomadtechie) [inline .travis.yml]: @bmac some items to consider adding here:

```
+install:
+script:
+  - python3 build.py
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_TOKEN  # Set in the settings page of your repository, as a secure variable
+  keep-history: true
+  on:
+    branch: master
+  target-branch: gh-pages
```
Taken from https://github.com/bocoup/wpt-disabled-tests-report/commit/995475970e058c3504b10831bf8f304ad8e35e3f

Comment (bmac) [reply .travis.yml]: Thanks @nomadtechie. Looks like most of these are options for the github pages deployment provider so I don't think we need any of these options yet.

**Comment by bmac** [comment]
The cron job in this branch has successfully run twice, and pushed up 2 branches to this repo. 
screen shot 2018-06-21 at 8 28 22 am

The next run is scheduled for 2:15pm today.

**Pull request closed without merging**